virtcontainers/container.go

```diff
@@ -612,6 +612,24 @@ func (c *Container) unmountHostMounts() error {
 				return err
 			}
 
+			if m.Type == "bind" {
+				s, err := os.Stat(m.HostPath)
+				if err != nil {
+					c.Logger().WithFields(logrus.Fields{
+						"host-path": m.HostPath,
+						"error":     err,
```

Comment:
The error should be handled like this:

```go
c.Logger().WithError(err).WithFields(...)
```

But is this log call even required if the caller logs the error (I haven't checked)?

Reply:
> The error should be handled like this

I copied this log line from the prev codes in the func:
```golang
if err := syscall.Unmount(m.HostPath, syscall.MNT_DETACH); err != nil {
  c.Logger().WithFields(logrus.Fields{
    "host-path": m.HostPath,
    "error":     err,
  }).Warn("Could not umount")
  return err
```
So is it an old style and should be `WithError(err)` now?

> is this log call even required

I also wonder why does `syscall.Unmount` do such log here... maybe it includes the `m.HostPath` which isn't carried by the error?